Neos.Media/Classes/Domain/Repository/ImageVariantRepository.php

```diff
@@ -0,0 +1,96 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Media\Domain\Repository;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Media\Domain\Model\ImageVariant;
+use Neos\Media\Domain\ValueObject\Configuration\VariantPreset;
+
+/**
+ * A repository for ImageVariants
+ *
+ * @Flow\Scope("singleton")
+ */
+class ImageVariantRepository extends AssetRepository
+{
+    public function findVariantsByName(?string $identifier, ?string $variantName, int $limit = null): array
```

Comment:
hold my beer, why are the parameters ?nullable

Reply:
its not used with nullable params, so we can remove the nullablity and also below: `if (is_null($identifier) && is_null($variantName)) {`